certbot/cli.py

```diff
@@ -709,6 +709,11 @@ def prepare_and_parse_args(plugins, args, detect_defaults=False):  # pylint: dis
         help="With the register verb, indicates that details associated "
              "with an existing registration, such as the e-mail address, "
              "should be updated, rather than registering a new account.")
+    helpful.add(
+        "register", "--deactivate", action="store_true",
+        help="Irrevocably deactivate your account. Certificates associated "
+             "with your account can still be revoked, but they can not be "
+             "renewed.")
```

Comment:
Strange, I didn't know boulder would allow you to do that. If you can still revoke you certs and still renew you certs, then what is even the point of account deactivation?  I thought the point was to block someone from renewing your certs if your account was compromised.